qmlti/lib.php

```diff
@@ -217,9 +218,47 @@ function init_db($db) {
   [result_sourcedid] VARCHAR(255),
   [score] VARCHAR(255),
   [created] DATETIME,
+  [report_url] VARCHAR(255),
  CONSTRAINT [PK_{$outcome_table_name}] PRIMARY KEY CLUSTERED ([result_sourcedid] ASC, [score] ASC, [created] ASC)
 )
 EOD;
+ 
+//        $ok = $db->exec($sql) !== FALSE;
+        $db->exec($sql);
+
+      }
+
+      $reports_table_name = TABLE_PREFIX . LTI_Data_Connector::REPORTS_TABLE_NAME;
+      if ($ok && !sqlsrv_table_exists($db, $reports_table_name)) {
+
+        $sql = <<< EOD
+CREATE TABLE [dbo].[{$reports_table_name}] (
+  [context_id] VARCHAR(255),
+  [assessment_id] VARCHAR(255),
+  [is_accessible] BIT,
+ CONSTRAINT [PK_{$reports_table_name}] PRIMARY KEY CLUSTERED ([context_id] ASC, [assessment_id] ASC, [is_accessible] ASC)
```

Comment:
I don't think you primary key makes sense. is_accessible is a property of the assessment right? 

would you really want both of these to exist:
"Physics 101, "AstroPhysics", true 
"Physics 101, "AstroPhysics", false